app/main.py

```diff
@@ -1,8 +1,56 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self
+                 , comfort_class: int
+                 , clean_mark: int
+                 , brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self
+                 , distance_from_city_center: float
+                 , clean_power: int
+                 , average_rating: float
+                 , count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> None:
```

Comment:
The `calculate_washing_price` method is defined to return `None`, but it should return the calculated price. Change the return type to `float` or remove the `-> None` annotation.